central/networkgraph/entity/datastore/datastore_impl_test.go

```diff
@@ -226,6 +226,11 @@ func (suite *NetworkEntityDataStoreTestSuite) TestNetworkEntities() {
 	suite.NoError(err)
 	suite.Len(entities, 3)
 
+	query = search.NewQueryBuilder().AddStrings(search.ExternalSourceAddress, "192.0.2.0/29").ProtoQuery()
```

Comment:
nit:  I recommend a test where you find nothing.  Just to be safe that the query builder and queries are doing what you expect them to do.

Reply:
Done